src/main/java/mate/academy/spring/boot/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.spring.boot.controller;
+
+import java.util.List;
+import mate.academy.spring.boot.dto.BookDto;
+import mate.academy.spring.boot.dto.CreateBookRequestDto;
+import mate.academy.spring.boot.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
```

Comment:
```suggestion
@RequestMapping("/books")
```

Add `server.servlet.context-path=/api` in application.properties